docs/resources/aws_ecr.md

```diff
@@ -3,6 +3,13 @@ title: About the aws_ecr Resource
 platform: aws
 ---
 
+> WARNING: **This resource is deprecated. Please use following resources instead.** 
```

Comment:
Perhaps this could be updated to `WARNING: **This resource is deprecated. Please use one of the following resources instead.** `